src/SOS/Strike/util.cpp

```diff
@@ -5487,7 +5489,7 @@ GetLineByOffset(
     IfFailRet(ConvertNativeToIlOffset(offset, &pModule, &methodToken, &methodOffs));
 
     ToRelease<IMetaDataImport> pMDImport(NULL);
-    IfFailRet(pModule->QueryInterface(IID_IMetaDataImport, (LPVOID *) &pMDImport));
+    pModule->QueryInterface(IID_IMetaDataImport, (LPVOID *) &pMDImport);
```

Comment:
Can the SymbolReader achieve much without the proper MDImport?

Reply:
Yes, it can look for/download a portable PDB. The MD import is only required for the Windows PDB path. This change was to make source/line info work better for triage dumps (which sometimes can't get the mdimport from the module).